src/Native/configure.cmake

```diff
@@ -235,10 +249,17 @@ check_cxx_source_runs(
 check_prototype_definition(
     getpriority
     "int getpriority(int which, int who)"
-    "0"
+    0
```

Comment:
I'm not clear on this change either.  @jasonwilliams200OK, can you elaborate on why it was necessary?  The example in the cmake docs (https://cmake.org/cmake/help/v3.3/module/CheckPrototypeDefinition.html) shows NULL surrounded by quotes in a similar situation.  Are the quotes harmful here for some reason?